.all-contributorsrc

```diff
@@ -205,6 +205,13 @@
       "contributions": [
         "tool"
       ]
+    },
+    {
+      "login": "thomhos",
+      "name": "thom",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/11661846?v=4",
+      "profile": "http://thom.kr",
+      "contributions": []
```

Comment:
Mind adding me too (if it's okay with @kentcdodds)? I had forgotten to add myself to this list. 😃

Reply:
done!